CHANGELOG.md

```diff
@@ -1,4 +1,36 @@
 # Changelog
+## [1.3.0] - 2021-03-24
+### Changes
+- Use inclusive language
+Use inclusive language in docs, variable names, function names and parameter names
```

Comment:
This is repeating `Use inclusive language`, mentioning "docs, variable names, ..." is too much detail in my opition.

Reply:
I changed into 
```
- Use inclusive language
  - "slave" is deprecated in favor of "port"
  - "master" is deprecated in favor of "controller"
  ```